server.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const logger = require("morgan");
+const cors = require("cors");
+const db = require("./db");
+const routes = require("./routes");
+
+const app = express();
+const PORT = process.env.PORT || 4567
+//Middleware
+app.use(cors());
+app.use(express.jason());
+app.use(logger("dev"));
+//
+app.use()
```

Comment:
add "  '/api', routes " to the () of app.use

Reply:
Done